src/controllers/index.js

```diff
@@ -0,0 +1,22 @@
+
+/**
+ * class Root defines root apis
+ * @class
+ */
+class Root {
+  /**
+   * @param {Oject} req is the request object
+   * @param {Object} res is the response object
+   * @static
+   * @returns {void}
+   */
+  static getRoot(req, res) {
+    res.status(201).send({
```

Comment:
Why 201 status? Is anything being created?

Reply:
it has been updated. thanks for the feedback